test/functional/test_framework/util.py

```diff
@@ -244,14 +244,13 @@ class PortSeed:
     # Must be initialized with a unique integer for each process
     n = None
 
-def get_rpc_proxy(url, node_number, timeout=None, coveragedir=None):
+def get_rpc_proxy(url, node_number, *, timeout=None, coveragedir=None):
     """
     Args:
         url (str): URL of the RPC server to call
         node_number (int): the node number (or id) that this calls to
-
-    Kwargs:
```

Comment:
Any reason to remove "Kwargs" here? Especially with the `*, ` it seems important to mention that these need to be keyword arguments.

Reply:
I put `Kwargs:` back now.